pkg/registry/registry.go

```diff
@@ -152,3 +162,48 @@ func GetRegistryAndRepo(tag string) (string, string) {
 	}
 	return registryTag, imageTag
 }
+
+func GetHiddenExposePorts(image string) []int32 {
+	exposedPorts := make([]int32, 0)
+	c, err := NewRegistryClient("https://registry.hub.docker.com", "", "")
```

Comment:
what if the image is not in docker hub? (e.g. in our own)

Reply:
For now it will only inspect docker hub public images. I think this will cover most of the compose files. When using their own image the user usually set the ports in the expose field. This port injection is costly, so it is good to perform it as few times as possible.

Reply:
Would this work with images with these formats?
- docker.io/bitnami/mongodb
- mongodb

Reply:
New commits adds support for public images from other repositories. It also adds support for private images on our own registry.

Reply:
This will work with both formats. It will have issues when the image is a official one and adds the registry. For example, `docker.io/mongo`.